README.md

```diff
@@ -57,26 +57,24 @@ Click on a link below to try Elyra, on a sandbox environment, without having to
 
 #### Using Docker
 
-You can also try Elyra by running one of the docker images from [Docker Hub](https://hub.docker.com/r/elyra/elyra/tags):
+You can also try Elyra by running one of the container images from [Docker Hub](https://hub.docker.com/r/elyra/elyra/tags) or [quay.io](https://quay.io/repository/elyra/elyra?tab=tags):
 - `elyra/elyra:latest` has the latest released version installed.
 - `elyra/elyra:x.y.z` has a specific version installed.
-- `elyra/elyra:dev` is automatically re-built each time a change is committed to the main branch.
 
-The command below starts the most recent development build in a clean environment:
+The command below starts the most recent official release in a clean environment:
```

Comment:
No, this is a readme for who is reading it from the main branch and should be dev. When you go to a given tag, this gets updated to the release it is pointing to on that tag, see https://github.com/elyra-ai/elyra/blob/v3.10.1/README.md.

I would be ok to add `latest` option, but I don't think we should remove `dev` as anyone working on the contribution side would try to build and that would be the way to use it. 

Also, changes here need to be reflected in the release script.

Reply:
I addressed your review comments.

Reply:
I think we're at a bit of a stalemate with this in deciding whether this is meant for devs vs end users. 
Can we maybe just split down the middle and keep the references to the `dev` tag for the docker image in another doc (if it doesnt exist already) and add a link in this section further down?

Reply:
I reverted the relevant changes and added/changed wording to include information about `dev` images.